### PR TITLE
Added test case

### PR DESCRIPTION
**Pull request by ColinHarker**
Still trying to learn the basics of this library, made a small test case just to see if this is proper format before I dive in deeper.

### CODE REVIEW

**Comment by ColinH** [comment]
The travis-ci build failed only due to your code not being formatted with clang-format according to our style, that can easily be fixed.

The AppVeyor build failed because unfortunately CMake, which is the basis for our Windows builds, requires all `.cpp` files in `src/test/json` to be listed in `src/test/json/CMakeLists.txt`, can you add your new test there?

**Comment by ColinHarker** [comment]
Ah okay, I just added the file to the cmakelist.

File: src/test/json/events_non_finite_to.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#include <tao/json/events/non_finite_to_null.hpp>
+#include <tao/json/events/non_finite_to_string.hpp>
+
+#include <tao/json.hpp>
+
+namespace tao::json
+{
+   value test_non_finite_to_string( const value& v, const std::string& s )
+   {
+      return s.compare( to_string< events::non_finite_to_string >( v ) );
```
Comment (d-frey) [inline src/test/json/events_non_finite_to.cpp:17]: Why not `to_string<...>(v) == s`? This is C++, not Java ;)

Comment (ColinHarker) [reply src/test/json/events_non_finite_to.cpp:17]: I'll make the change! gotta get out of old habits haha.

**Comment by d-frey** [comment]
Are you still working on this? It is still marked as a Draft.

**Pull request closed without merging**